### PR TITLE
Add missing autoCorrect declarations

### LINKED ISSUE #1796: Docs: Missing autoCorrect option for various rules
It seems that ktlint [supports autoCorrect for indentation rule](https://github.com/pinterest/ktlint/blob/88c9e80d973dd7932454ef086c532305a492fe07/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/IndentationRule.kt#L24) but it is [not declared on detekt side](https://github.com/arturbosch/detekt/blob/f98ccac9aa352805621dd6a19888bb472d776cef/detekt-formatting/src/main/kotlin/io/gitlab/arturbosch/detekt/formatting/wrappers/Indentation.kt#L14-L15).

Could it be added so that we can benefit from that? I didn't check other rules, maybe some others are missing as well.

**edit: Seems to be missing for:**

- [x] IndentationRule
- [x] AnnotationOnSeparateLine (AnnotationRule)
- [ ] ~~MaximumLineLength~~
- [ ] ~~NoWildcardImports~~
- [ ] ~~Filename~~
- [x] ImportOrdering

### PR DESCRIPTION
**Pull request by bagage**
Fixes #1796

### CODE REVIEW

File: detekt-cli/src/main/resources/default-detekt-config.yml [hunk 1]
```diff
@@ -242,6 +247,7 @@ formatting:
     autoCorrect: true
   NoWildcardImports:
     active: true
+    autoCorrect: true
```
Comment (schalkms) [inline detekt-cli/src/main/resources/default-detekt-config.yml]: This rule has no `autoCorrect` feature. Please take a look at the [source](https://github.com/pinterest/ktlint/blob/master/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/NoWildcardImportsRule.kt).

File: detekt-cli/src/main/resources/default-detekt-config.yml [hunk 2]
```diff
@@ -192,17 +193,21 @@ formatting:
...
     active: false
+    autoCorrect: true
   Indentation:
     active: false
+    autoCorrect: true
     indentSize: 4
     continuationIndentSize: 4
   MaximumLineLength:
     active: true
+    autoCorrect: true
```
Comment (schalkms) [inline detekt-cli/src/main/resources/default-detekt-config.yml]: This rule has no `autoCorrect` feature. Please take a look at the [source](https://github.com/pinterest/ktlint/blob/master/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/MaxLineLengthRule.kt).

File: detekt-cli/src/main/resources/default-detekt-config.yml [hunk 3]
```diff
@@ -192,17 +193,21 @@ formatting:
     autoCorrect: true
   Filename:
     active: true
+    autoCorrect: true
```
Comment (schalkms) [inline detekt-cli/src/main/resources/default-detekt-config.yml]: This rule has no `autoCorrect` feature. Please take a look at the [source](https://github.com/pinterest/ktlint/blob/master/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/FilenameRule.kt.

File: detekt-cli/src/main/resources/default-detekt-config.yml (hunk 1 above)
Comment (bagage) [reply detekt-cli/src/main/resources/default-detekt-config.yml]: Indeed, I misinterpreted ktlint sourcecode! Thanks you.

File: detekt-cli/src/main/resources/default-detekt-config.yml (hunk 3 above)
Comment (bagage) [reply detekt-cli/src/main/resources/default-detekt-config.yml]: Indeed, I misinterpreted ktlint sourcecode! Thanks you.

File: detekt-cli/src/main/resources/default-detekt-config.yml (hunk 2 above)
Comment (bagage) [reply detekt-cli/src/main/resources/default-detekt-config.yml]: Indeed, I misinterpreted ktlint sourcecode! Thanks you.

**Comment by bagage** [comment]
Should be ok now :tada:

**Pull request merged**